### PR TITLE
New article: Set up office 365

### PR DESCRIPTION
**Pull request by DonNateR**
- Also includes changes to the Office 365 introduction page

### CODE REVIEW

**Comment by rackernexus** [comment]
:mag_right: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-df66bf9801/how-to/)

**Comment by rackernexus** [comment]
If you want to double-check your formatting, check [the preview](https://staging.developer.rackspace.com/support.rackspace.com/build-54907fc175/how-to/).

**Comment by kmsholcomb** [comment]
Hi @DonNateR. Edits for both articles attached. Let me know if you have any questions. Thanks!

[Microsoft Office 365_editsKH.docx](https://github.com/rackerlabs/rackspace-how-to/files/1132237/Microsoft.Office.365_editsKH.docx)
[Microsoft Office 365_editsKH.pdf](https://github.com/rackerlabs/rackspace-how-to/files/1132236/Microsoft.Office.365_editsKH.pdf)

[Set up Office 365_editsKH.docx](https://github.com/rackerlabs/rackspace-how-to/files/1132234/Set.up.Office.365_editsKH.docx)
[Set up Office 365_editsKH.pdf](https://github.com/rackerlabs/rackspace-how-to/files/1132235/Set.up.Office.365_editsKH.pdf)

**Comment by rackernexus** [comment]
:mag_right: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-192282372a/how-to/)

**Comment by rackernexus** [comment]
Your content preview [is now ready](https://staging.developer.rackspace.com/support.rackspace.com/build-b0d596015c/how-to/). :bow:

**Comment by rackernexus** [comment]
:eyes: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-dc849cbb2d/how-to/)

**Comment by rackernexus** [comment]
Your content preview [is now ready](https://staging.developer.rackspace.com/support.rackspace.com/build-5297b1b183/how-to/). :bow:

File: content/office-365/set-up-office-365.md [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+
+The tenant ID that you create is permanent. If you have an existing tenant ID for your Office 365 account, contact Microsoft Support to remove it. Removing the existing ID enables you to create a new ID for your account.
+
+1. Log in to the [Cloud Office Control Panel](https://cp.rackspace.com/) using your Rackspace Cloud Office admin ID and password.
+2. In the Microsoft Office 365 section, select **Manage Office 365**.
+3. Enter a tenant name using only letters and numbers.
+
+   A tenant is a unique name for your organization within the Office 365 portal. Your tenant name forms your Office 365 user ID. A tenant is not a domain.
+
+   If the tenant ID that you enter is valid and available, a green check box appears next to the name
```
Comment (stephamon) [inline content/office-365/set-up-office-365.md]: Missing period at end of sentence.

**Comment by rackernexus** [comment]
Your content preview [is now ready](https://staging.developer.rackspace.com/support.rackspace.com/build-1691d55dd8/how-to/). :bow:

File: content/office-365/set-up-office-365.md [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+
+4. Enter values in the following fields. All fields are required.
+
+   - **User name**: Enter a unique user name.
+   - **Password**: Enter a unique password. This password is temporary The Office 365 portal prompts you to create a new password when you first visit the site.
```
Comment (stephamon) [inline content/office-365/set-up-office-365.md]: Missing period after "temporary."

File: content/office-365/set-up-office-365.md [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+   - **Company information**: Enter the information for your company.
+
+5. Click **Create Tenant**.
+
+The tenant name, user name, and password that you enter in Cloud Office form your Office 365 user ID. Your SharePoint (**<tenantName>.sharepoint.com**) and OneDrive (**<tenantName>-my.sharepoint.com**) URLs use the tenant ID for the life of your account.
+
+### Add a subscription
+
+After you create an Office 365 tenant ID, chose an Office 365 subscription. To compare Office 365 subscriptions offered by Rackspace, see [Office 365 - Pick your plan](https://www.rackspace.com/office-365/pick-your-plan).
```
Comment (stephamon) [inline content/office-365/set-up-office-365.md]: "chose" should be "choose"

**Comment by rackernexus** [comment]
If you want to double-check your formatting, check [the preview](https://staging.developer.rackspace.com/support.rackspace.com/build-3183542cf1/how-to/).

**Comment by stephamon** [comment]
👍

**Pull request merged**